PurchasesCoreSwift/Logging/LoggerWrapper.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  LoggerWrapper.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/08/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCLogIntent) public enum LogIntent: Int {
```

Comment:
do we intend to use this directly from objective-c? if not, the class shouldn't be public (internal is the default), and the `@objc` can be ommitted

Reply:
we might also wanna have a separate file for it, we mostly try to keep 1 file per class